src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,13 +1,30 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+    private String name;
+    private String color;
+    private String type;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator` class. Fields should be private to follow encapsulation principle.